packages/material/src/complex/MaterialObjectRenderer.tsx

```diff
@@ -47,23 +47,22 @@ export const MaterialObjectRenderer = ({
   uischema,
   rootSchema
 }: StatePropsOfControlWithDetail) => {
+
   const detailUiSchema = useMemo(
     () =>
       findUISchema(
         uischemas,
         schema,
         uischema.scope,
         path,
-        'Group',
+        ()=>Generate.uiSchema(schema, 'Group',undefined,undefined,label),
```

Comment:
I suggest a consistent use of spaces.
```suggestion
        () => Generate.uiSchema(schema, 'Group', undefined, undefined, label),
```